packages/headless/src/controllers/facets/facet/headless-facet.ts

```diff
@@ -96,15 +96,15 @@ export function buildFacet(
     ...controller,
     facetSearch: createFacetSearch(),
     /**
-     * Selects (deselects) the passed value if unselected (selected).
-     * @param selection The facet value to select or deselect.
+     * Toggles the passed value.
+     * @param selection (FacetValue) The facet value to select or deselect.
      */
     toggleSelect: (selection: FacetValue) =>
       dispatch(executeToggleFacetSelect({facetId: options.facetId, selection})),
     /**
      * Returns `true` if the passed facet value is selected and `false` otherwise.
-     * @param {FacetValue} The facet value to check.
-     * @returns {boolean}.
+     * @param value (FacetValue) The facet value to check.
+     * @returns Whether the passed facet value is selected.
```

Comment:
Just a reminder: I believe we agreed to also specify the return type for return annotations 🙂  Be sure to check this throughout your changes